source/adminguide/accounts.rst

```diff
@@ -279,17 +279,99 @@ or ApacheDS to authenticate CloudStack end-users. CloudStack will search
 the external LDAP directory tree starting at a specified base directory
 and gets user info such as first name, last name, email and username.
 
-Starting with CloudStack 4.11, an ldap connection per domain can be
-defined.
+Starting with CloudStack 4.11, an LDAP connection per domain can be
+defined. In this domain autosync per account can be configured,
+keeping the users in the domain up to date with their group membership
+in LDAP.
+.. Note:: A caveat with this is that ApacheDS does not yet support the
```

Comment:
There should be a newline before this, or it will flow into the previous paragraph.

Reply:
resolved